microprofile-jwt/src/main/java/org/jboss/eap/qe/microprofile/jwt/testapp/Roles.java

```diff
@@ -0,0 +1,11 @@
+package org.jboss.eap.qe.microprofile.jwt.testapp;
```

Comment:
testapp ?

use more descriptive package name

Reply:
Thank you! I chose this name because the package contains a testing application code. Do you suggest some other name?